Index: OgreMain/include/OgreMovableObject.h
===================================================================
--- OgreMain/include/OgreMovableObject.h	(revision 9195)
+++ OgreMain/include/OgreMovableObject.h	(revision 9196)
@@ -322,7 +322,8 @@
             a 3D menu system or such.
         @par
             See RenderQueue for more details.
-        @param queueID Enumerated value of the queue group to use.
+        @param queueID Enumerated value of the queue group to use. See the
+			enum RenderQueueGroupID for what kind of values can be used here.
         */
         virtual void setRenderQueueGroup(uint8 queueID);
 
Index: OgreMain/include/OgreRenderQueue.h
===================================================================
--- OgreMain/include/OgreRenderQueue.h	(revision 9195)
+++ OgreMain/include/OgreRenderQueue.h	(revision 9196)
@@ -225,7 +225,8 @@
         ushort getDefaultRenderablePriority(void) const;
 
         /** Sets the current default queue group, which will be used for all renderable which do not
-            specify which group they wish to be on.
+            specify which group they wish to be on. See the enum RenderQueueGroupID for what kind of
+			values can be used here.
         */
         void setDefaultQueueGroup(uint8 grp);
         
