Index: RenderSystems/GL/src/OgreGLRenderSystem.cpp
===================================================================
--- RenderSystems/GL/src/OgreGLRenderSystem.cpp	(revision 9201)
+++ RenderSystems/GL/src/OgreGLRenderSystem.cpp	(revision 9202)
@@ -3259,6 +3259,17 @@
 		if (mCurrentFragmentProgram)
 			mCurrentFragmentProgram->unbindProgram();
 
+		// Disable lights
+		for (unsigned short i = 0; i < mCurrentLights; ++i)
+		{
+			setGLLight(i, NULL);
+			mLights[i] = NULL;
+		}
+		mCurrentLights = 0;
+
+		// Disable textures
+		_disableTextureUnitsFrom(0);
+
 		// It's ready to switching
 		if (mCurrentContext)
 			mCurrentContext->endCurrent();
