diff -up ogre-1.7.2/CMake/Templates/OGRE-Paging.pc.in.BAD ogre-1.7.2/CMake/Templates/OGRE-Paging.pc.in
--- ogre-1.7.2/CMake/Templates/OGRE-Paging.pc.in.BAD	2010-12-21 17:40:45.478242000 -0500
+++ ogre-1.7.2/CMake/Templates/OGRE-Paging.pc.in	2010-12-21 17:40:51.225241999 -0500
@@ -1,6 +1,6 @@
 prefix=@OGRE_PREFIX_PATH@
 exec_prefix=${prefix}
-libdir=${prefix}/@OGRE_LIB_DIRECTORY@
+libdir=@OGRE_LIB_DIRECTORY@
 includedir=${prefix}/include
 
 Name: OGRE-Paging
diff -up ogre-1.7.2/CMake/Templates/OGRE.pc.in.BAD ogre-1.7.2/CMake/Templates/OGRE.pc.in
--- ogre-1.7.2/CMake/Templates/OGRE.pc.in.BAD	2010-12-21 17:40:19.968241999 -0500
+++ ogre-1.7.2/CMake/Templates/OGRE.pc.in	2010-12-21 17:40:31.873241999 -0500
@@ -1,6 +1,6 @@
 prefix=@OGRE_PREFIX_PATH@
 exec_prefix=${prefix}
-libdir=${prefix}/@OGRE_LIB_DIRECTORY@
+libdir=@OGRE_LIB_DIRECTORY@
 includedir=${prefix}/include
 plugindir=${libdir}/OGRE
 
diff -up ogre-1.7.2/CMake/Templates/OGRE-PCZ.pc.in.BAD ogre-1.7.2/CMake/Templates/OGRE-PCZ.pc.in
--- ogre-1.7.2/CMake/Templates/OGRE-PCZ.pc.in.BAD	2010-12-21 17:41:02.368242000 -0500
+++ ogre-1.7.2/CMake/Templates/OGRE-PCZ.pc.in	2010-12-21 17:41:07.937242000 -0500
@@ -1,6 +1,6 @@
 prefix=@OGRE_PREFIX_PATH@
 exec_prefix=${prefix}
-libdir=${prefix}/@OGRE_LIB_DIRECTORY@
+libdir=@OGRE_LIB_DIRECTORY@
 includedir=${prefix}/include
 
 Name: OGRE-PCZ
diff -up ogre-1.7.2/CMake/Templates/OGRE-Property.pc.in.BAD ogre-1.7.2/CMake/Templates/OGRE-Property.pc.in
--- ogre-1.7.2/CMake/Templates/OGRE-Property.pc.in.BAD	2010-12-21 17:41:14.796241999 -0500
+++ ogre-1.7.2/CMake/Templates/OGRE-Property.pc.in	2010-12-21 17:41:19.865242000 -0500
@@ -1,6 +1,6 @@
 prefix=@OGRE_PREFIX_PATH@
 exec_prefix=${prefix}
-libdir=${prefix}/@OGRE_LIB_DIRECTORY@
+libdir=@OGRE_LIB_DIRECTORY@
 includedir=${prefix}/include
 
 Name: OGRE-Property
diff -up ogre-1.7.2/CMake/Templates/OGRE-RTShaderSystem.pc.in.BAD ogre-1.7.2/CMake/Templates/OGRE-RTShaderSystem.pc.in
--- ogre-1.7.2/CMake/Templates/OGRE-RTShaderSystem.pc.in.BAD	2010-12-21 17:41:27.599242000 -0500
+++ ogre-1.7.2/CMake/Templates/OGRE-RTShaderSystem.pc.in	2010-12-21 17:41:33.593241999 -0500
@@ -1,6 +1,6 @@
 prefix=@OGRE_PREFIX_PATH@
 exec_prefix=${prefix}
-libdir=${prefix}/@OGRE_LIB_DIRECTORY@
+libdir=@OGRE_LIB_DIRECTORY@
 includedir=${prefix}/include
 
 Name: OGRE-RTShaderSystem
diff -up ogre-1.7.2/CMake/Templates/OGREStatic.pc.in.BAD ogre-1.7.2/CMake/Templates/OGREStatic.pc.in
--- ogre-1.7.2/CMake/Templates/OGREStatic.pc.in.BAD	2010-12-21 17:41:45.839242000 -0500
+++ ogre-1.7.2/CMake/Templates/OGREStatic.pc.in	2010-12-21 17:41:50.988242000 -0500
@@ -1,6 +1,6 @@
 prefix=@OGRE_PREFIX_PATH@
 exec_prefix=${prefix}
-libdir=${prefix}/@OGRE_LIB_DIRECTORY@
+libdir=@OGRE_LIB_DIRECTORY@
 includedir=${prefix}/include
 plugindir=${libdir}/OGRE
 
diff -up ogre-1.7.2/CMake/Templates/OGRE-Terrain.pc.in.BAD ogre-1.7.2/CMake/Templates/OGRE-Terrain.pc.in
--- ogre-1.7.2/CMake/Templates/OGRE-Terrain.pc.in.BAD	2010-12-21 17:41:55.667242000 -0500
+++ ogre-1.7.2/CMake/Templates/OGRE-Terrain.pc.in	2010-12-21 17:42:01.556241999 -0500
@@ -1,6 +1,6 @@
 prefix=@OGRE_PREFIX_PATH@
 exec_prefix=${prefix}
-libdir=${prefix}/@OGRE_LIB_DIRECTORY@
+libdir=@OGRE_LIB_DIRECTORY@
 includedir=${prefix}/include
 
 Name: OGRE-Terrain
