diff -ur ogrenew/PlatformManagers/SDL/src/OgreSDLConfig_gtk.cpp ogrenew.new/PlatformManagers/SDL/src/OgreSDLConfig_gtk.cpp
--- ogrenew/PlatformManagers/SDL/src/OgreSDLConfig_gtk.cpp	2006-01-26 13:38:24.000000000 +0100
+++ ogrenew.new/PlatformManagers/SDL/src/OgreSDLConfig_gtk.cpp	2006-06-16 22:30:00.000000000 +0200
@@ -23,6 +23,7 @@
 */
 
 #include "OgreSDLConfig.h"
+#include "OgreLogManager.h"
 
 #include <libglademm/xml.h>
 
