Index: Samples/Shadows/src/Shadows.cpp
===================================================================
--- Samples/Shadows/src/Shadows.cpp	(revision 9212)
+++ Samples/Shadows/src/Shadows.cpp	(revision 9213)
@@ -1388,7 +1388,7 @@
 				switch(mat)
 				{
 				case MAT_STANDARD:
-					mSceneMgr->setShadowTexturePixelFormat(PF_L8);
+					mSceneMgr->setShadowTexturePixelFormat(PF_X8R8G8B8);
 					mSceneMgr->setShadowTextureCasterMaterial(StringUtil::BLANK);
 					mSceneMgr->setShadowTextureReceiverMaterial(StringUtil::BLANK);
 					mSceneMgr->setShadowTextureSelfShadow(false);	
