Index: RenderSystems/Direct3D9/src/OgreD3D9RenderWindow.cpp
===================================================================
--- RenderSystems/Direct3D9/src/OgreD3D9RenderWindow.cpp	(revision 9336)
+++ RenderSystems/Direct3D9/src/OgreD3D9RenderWindow.cpp	(revision 9337)
@@ -779,7 +779,7 @@
 			}
 			else
 			{
-				hr = mpD3DDevice->Present( NULL, NULL, 0, NULL );
+				hr = mpD3DDevice->Present( NULL, NULL, NULL, NULL );
 			}
 			if( D3DERR_DEVICELOST == hr )
 			{
