Index: OgreMain/src/OgreEdgeListBuilder.cpp
===================================================================
--- OgreMain/src/OgreEdgeListBuilder.cpp	(revision 9341)
+++ OgreMain/src/OgreEdgeListBuilder.cpp	(revision 9342)
@@ -432,11 +432,14 @@
         assert(triangleFaceNormals.size() == triangleLightFacings.size());
 
         // Use optimised util to determine if triangle's face normal are light facing
-        OptimisedUtil::getImplementation()->calculateLightFacing(
-            lightPos,
-            &triangleFaceNormals.front(),
-            &triangleLightFacings.front(),
-            triangleLightFacings.size());
+		if(!triangleFaceNormals.empty())
+		{
+			OptimisedUtil::getImplementation()->calculateLightFacing(
+				lightPos,
+				&triangleFaceNormals.front(),
+				&triangleLightFacings.front(),
+				triangleLightFacings.size());
+		}
     }
     //---------------------------------------------------------------------
     void EdgeData::updateFaceNormals(size_t vertexSet, 
