--- ./OgreMain/src/OgreDynLib.cpp.orig	2020-04-26 00:11:59.000000000 +0100
+++ ./OgreMain/src/OgreDynLib.cpp	2020-05-29 15:30:38.474355025 +0100
@@ -80,7 +80,9 @@ namespace Ogre {
             name += ".js";
 #elif OGRE_PLATFORM == OGRE_PLATFORM_LINUX
         // dlopen() does not add .so to the filename, like windows does for .dll
-        if (name.find(".so") == String::npos)
+        mInst = (DYNLIB_HANDLE)DYNLIB_LOAD( name.c_str() );
+        // Only add .so if open without it failed
+        if ((!mInst) && (name.find(".so") == String::npos))
         {
             name += StringUtil::format(".so.%d.%d.%d", OGRE_VERSION_MAJOR, OGRE_VERSION_MINOR, OGRE_VERSION_PATCH);
         }
@@ -98,7 +100,7 @@ namespace Ogre {
         // Log library load
         LogManager::getSingleton().logMessage("Loading library " + name);
 
-        mInst = (DYNLIB_HANDLE)DYNLIB_LOAD( name.c_str() );
+        if( !mInst ) mInst = (DYNLIB_HANDLE)DYNLIB_LOAD( name.c_str() );
 #if OGRE_PLATFORM == OGRE_PLATFORM_APPLE
         if(!mInst)
         {
