--- ogrenew/OgreMain/src/Makefile.in~	2006-06-15 23:31:11.000000000 +0200
+++ ogrenew/OgreMain/src/Makefile.in	2006-06-15 23:31:11.000000000 +0200
@@ -452,7 +452,7 @@
 	OgreCompiler2Pass.cpp $(am__append_1)
 platformdir = $(libdir)
 libOgreMain_la_LIBADD = $(STLPORT_LIBS) $(FT2_LIBS) $(ZZIPLIB_LIBS) $(OGRE_THREAD_LIBS)
-libOgreMain_la_LDFLAGS = $(SHARED_FLAGS) -version-info @OGREMAIN_VERSION_INFO@ -Wl,-rpath,$(platformdir)
+libOgreMain_la_LDFLAGS = $(SHARED_FLAGS) -version-info @OGREMAIN_VERSION_INFO@
 all: all-am
 
 .SUFFIXES:
--- ogrenew/OgreMain/src/Makefile.am~	2006-06-15 23:31:02.000000000 +0200
+++ ogrenew/OgreMain/src/Makefile.am	2006-06-15 23:31:02.000000000 +0200
@@ -174,6 +174,6 @@
 
 platformdir = $(libdir)
 libOgreMain_la_LIBADD= $(STLPORT_LIBS) $(FT2_LIBS) $(ZZIPLIB_LIBS) $(OGRE_THREAD_LIBS)
-libOgreMain_la_LDFLAGS = $(SHARED_FLAGS) -version-info @OGREMAIN_VERSION_INFO@ -Wl,-rpath,$(platformdir)
+libOgreMain_la_LDFLAGS = $(SHARED_FLAGS) -version-info @OGREMAIN_VERSION_INFO@
 #-Wl,--version-script=$(top_srcdir)/OgreMain/src/ogremain.map
 
