diff -Naur ogre-1.6.0rc1-clean/PlugIns/EXRCodec/src/Makefile.am ogre-1.6.0rc1-clean.new/PlugIns/EXRCodec/src/Makefile.am
--- ogre-1.6.0rc1-clean/PlugIns/EXRCodec/src/Makefile.am	2008-06-12 18:10:57.000000000 +0400
+++ ogre-1.6.0rc1-clean.new/PlugIns/EXRCodec/src/Makefile.am	2008-09-13 19:56:20.000000000 +0400
@@ -3,8 +3,7 @@
 pkglib_LTLIBRARIES = Plugin_EXRCodec.la
 
 Plugin_EXRCodec_la_SOURCES = OgreEXRCodec.cpp \
-			     O_IStream.cpp \
-                             OgreEXRCodecDll.cpp 
+			     O_IStream.cpp
 
 Plugin_EXRCodec_la_LDFLAGS = -module $(PLUGIN_FLAGS) -L$(top_builddir)/OgreMain/src -Wl,-z,defs
 Plugin_EXRCodec_la_LIBADD = -lOgreMain $(OPENEXR_LIBS) $(OGRE_THREAD_LIBS)
diff -Naur ogre-1.6.0rc1-clean/PlugIns/EXRCodec/src/Makefile.in ogre-1.6.0rc1-clean.new/PlugIns/EXRCodec/src/Makefile.in
--- ogre-1.6.0rc1-clean/PlugIns/EXRCodec/src/Makefile.in	2008-08-28 14:41:24.000000000 +0400
+++ ogre-1.6.0rc1-clean.new/PlugIns/EXRCodec/src/Makefile.in	2008-09-13 19:56:34.000000000 +0400
@@ -61,8 +61,7 @@
 am__DEPENDENCIES_1 =
 Plugin_EXRCodec_la_DEPENDENCIES = $(am__DEPENDENCIES_1) \
 	$(am__DEPENDENCIES_1)
-am_Plugin_EXRCodec_la_OBJECTS = OgreEXRCodec.lo O_IStream.lo \
-	OgreEXRCodecDll.lo
+am_Plugin_EXRCodec_la_OBJECTS = OgreEXRCodec.lo O_IStream.lo
 Plugin_EXRCodec_la_OBJECTS = $(am_Plugin_EXRCodec_la_OBJECTS)
 DEFAULT_INCLUDES = -I. -I$(srcdir) -I$(top_builddir)/OgreMain/include
 depcomp = $(SHELL) $(top_srcdir)/depcomp
@@ -253,8 +252,7 @@
 INCLUDES = $(STLPORT_CFLAGS) -I$(top_srcdir)/PlugIns/EXRCodec/include -I$(top_srcdir)/OgreMain/include $(OPENEXR_CFLAGS)
 pkglib_LTLIBRARIES = Plugin_EXRCodec.la
 Plugin_EXRCodec_la_SOURCES = OgreEXRCodec.cpp \
-			     O_IStream.cpp \
-                             OgreEXRCodecDll.cpp 
+			     O_IStream.cpp
 
 Plugin_EXRCodec_la_LDFLAGS = -module $(PLUGIN_FLAGS) -L$(top_builddir)/OgreMain/src -Wl,-z,defs
 Plugin_EXRCodec_la_LIBADD = -lOgreMain $(OPENEXR_LIBS) $(OGRE_THREAD_LIBS)
@@ -329,7 +327,6 @@
 
 @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/O_IStream.Plo@am__quote@
 @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/OgreEXRCodec.Plo@am__quote@
-@AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/OgreEXRCodecDll.Plo@am__quote@
 
 .cpp.o:
 @am__fastdepCXX_TRUE@	if $(CXXCOMPILE) -MT $@ -MD -MP -MF "$(DEPDIR)/$*.Tpo" -c -o $@ $<; \
