--- ogrenew/RenderSystems/GL/src/Makefile.am~	2008-01-12 21:45:45.000000000 +0100
+++ ogrenew/RenderSystems/GL/src/Makefile.am	2008-01-12 21:45:45.000000000 +0100
@@ -32,8 +32,7 @@ RenderSystem_GL_la_SOURCES = OgreGLEngin
                              OgreGLFBORenderTexture.cpp \
                              OgreGLPBRenderTexture.cpp \
                              OgreGLFBOMultiRenderTarget.cpp \
-                             OgreGLFrameBufferObject.cpp \
-                             glew.cpp
+                             OgreGLFrameBufferObject.cpp
 
 RenderSystem_GL_la_LDFLAGS = -module $(PLUGIN_FLAGS) -L$(top_builddir)/OgreMain/src -Wl,-z,defs
 RenderSystem_GL_la_LIBADD = \
@@ -41,4 +40,4 @@ RenderSystem_GL_la_LIBADD = \
     $(top_builddir)/RenderSystems/GL/src/GLSL/src/libGLSL.la \
     $(top_builddir)/RenderSystems/GL/src/nvparse/libnvparse.la \
     $(top_builddir)/RenderSystems/GL/src/atifs/src/libatifs.la \
-    $(GL_LIBS) $(GLSUPPORT_LIBS) -lOgreMain $(OGRE_THREAD_LIBS)
+    $(GL_LIBS) $(GLSUPPORT_LIBS) -lOgreMain $(OGRE_THREAD_LIBS) -lGLEW
--- ogrenew/RenderSystems/GL/src/OgreGLRenderSystem.cpp~	2008-01-12 19:59:20.000000000 +0100
+++ ogrenew/RenderSystems/GL/src/OgreGLRenderSystem.cpp	2008-01-12 19:59:20.000000000 +0100
@@ -54,7 +54,6 @@ Torus Knot Software Ltd.s
 
 // Convenience macro from ARB_vertex_buffer_object spec
 #define VBO_BUFFER_OFFSET(i) ((char *)NULL + (i))
-GLenum glewContextInit (Ogre::GLSupport *glSupport);
 
 namespace Ogre {
 
@@ -218,8 +217,8 @@ namespace Ogre {
 		LogManager::getSingleton().logMessage("*** GL Renderer Started ***");
 		LogManager::getSingleton().logMessage("***************************");
 
-		// Get extension function pointers
-        glewContextInit(mGLSupport);
+        // Get extension function pointers
+        glewInit();
 
         // Check for hardware mipmapping support.
         if(GLEW_VERSION_1_4 || GLEW_SGIS_generate_mipmap)
--- ogrenew/RenderSystems/GL/src/Makefile.in~	2008-01-12 21:46:10.000000000 +0100
+++ ogrenew/RenderSystems/GL/src/Makefile.in	2008-01-12 21:46:10.000000000 +0100
@@ -75,8 +75,7 @@ am_RenderSystem_GL_la_OBJECTS = OgreGLEn
 	OgreGLDefaultHardwareBufferManager.lo OgreGLContext.lo \
 	OgreGLPixelFormat.lo OgreGLRenderTexture.lo \
 	OgreGLFBORenderTexture.lo OgreGLPBRenderTexture.lo \
-	OgreGLFBOMultiRenderTarget.lo OgreGLFrameBufferObject.lo \
-	glew.lo
+	OgreGLFBOMultiRenderTarget.lo OgreGLFrameBufferObject.lo
 RenderSystem_GL_la_OBJECTS = $(am_RenderSystem_GL_la_OBJECTS)
 DEFAULT_INCLUDES = -I. -I$(srcdir) -I$(top_builddir)/OgreMain/include
 depcomp = $(SHELL) $(top_srcdir)/depcomp
@@ -299,8 +298,7 @@ RenderSystem_GL_la_SOURCES = OgreGLEngin
                              OgreGLFBORenderTexture.cpp \
                              OgreGLPBRenderTexture.cpp \
                              OgreGLFBOMultiRenderTarget.cpp \
-                             OgreGLFrameBufferObject.cpp \
-                             glew.cpp
+                             OgreGLFrameBufferObject.cpp
 
 RenderSystem_GL_la_LDFLAGS = -module $(PLUGIN_FLAGS) -L$(top_builddir)/OgreMain/src -Wl,-z,defs
 RenderSystem_GL_la_LIBADD = \
@@ -308,7 +306,7 @@ RenderSystem_GL_la_LIBADD = \
     $(top_builddir)/RenderSystems/GL/src/GLSL/src/libGLSL.la \
     $(top_builddir)/RenderSystems/GL/src/nvparse/libnvparse.la \
     $(top_builddir)/RenderSystems/GL/src/atifs/src/libatifs.la \
-    $(GL_LIBS) $(GLSUPPORT_LIBS) -lOgreMain $(OGRE_THREAD_LIBS)
+    $(GL_LIBS) $(GLSUPPORT_LIBS) -lOgreMain $(OGRE_THREAD_LIBS) -lGLEW
 
 all: all-recursive
 
@@ -401,7 +399,6 @@ distclean-compile:
 @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/OgreGLSupport.Plo@am__quote@
 @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/OgreGLTexture.Plo@am__quote@
 @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/OgreGLTextureManager.Plo@am__quote@
-@AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/glew.Plo@am__quote@
 
 .cpp.o:
 @am__fastdepCXX_TRUE@	if $(CXXCOMPILE) -MT $@ -MD -MP -MF "$(DEPDIR)/$*.Tpo" -c -o $@ $<; \
